sympy/printing/lambdarepr.py

```diff
@@ -348,6 +348,13 @@ def _print_Float(self, e):
         args = str(tuple(map(int, e._mpf_)))
         return 'mpf(%s)' % args
 
+    def _print_uppergamma(self,e): #printer for the uppergamma function
+        exp="gammainc({0}, {1}, 6)".format(e.args[0], e.args[1])
+        return 'mpf(%s)' %(self._print(exp))
```

Comment:
This method currently returns `lambda x,y: (mpf(gammainc(x, y, 6)))`. If the value of `gammainc` is of type `mpf`, the extra cast is not necessary.